pkg/gadgets/trace/bind/tracer/tracer_test.go

```diff
@@ -0,0 +1,659 @@
+//go:build linux
+// +build linux
+
+// Copyright 2022 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tracer_test
+
+import (
+	"fmt"
+	"net"
+	"sort"
+	"testing"
+	"time"
+
+	utilstest "github.com/kinvolk/inspektor-gadget/pkg/gadgets/internal/test"
+	"github.com/kinvolk/inspektor-gadget/pkg/gadgets/trace/bind/tracer"
+	"github.com/kinvolk/inspektor-gadget/pkg/gadgets/trace/bind/types"
+	eventtypes "github.com/kinvolk/inspektor-gadget/pkg/types"
+	"golang.org/x/sys/unix"
+)
+
+func TestBindTracerCreate(t *testing.T) {
```

Comment:
@mauriciovasquezbernal perhaps you could watch https://youtu.be/tWn8RA_DEic, I think it's a great resource about structuring and naming your tests, I highly recommend it.

Reply:
Thanks a lot for the hint, I'll definitely watch it!